src/main/java/uk/gov/hmcts/cmc/claimstore/rules/CountyCourtJudgmentRule.java

```diff
@@ -64,4 +64,25 @@ private boolean isResponseAlreadySubmitted(Claim claim) {
     private boolean isCountyCourtJudgmentAlreadySubmitted(Claim claim) {
         return claim.getCountyCourtJudgment() != null;
     }
+
+    private boolean isCountyCourtJudgmentAlreadyRedetermined(Claim claim) {
+        return claim.getReDeterminationRequestedAt().isPresent();
+    }
+
+    public void assertRedeterminationCanBeRequestedOnCountyCourtJudgement(Claim claim) {
+        Objects.requireNonNull(claim, "claim object can not be null");
+
+        String externalId = claim.getExternalId();
+
+        if (!isCountyCourtJudgmentAlreadySubmitted(claim)) {
```

Comment:
Question, can a user not request for redetermination in case of settlement agreement?